app/controllers/people_controller.rb

```diff
@@ -68,7 +71,7 @@ def inactive
   end
 
   def index
-    @people = Person.active.select{|person| person.department&.name == 'CERT' || person.department&.name == 'Police'}
+    @people = Person.active.select{|person| person.department&.shortname == 'CERT' || person.department&.shortname == 'BAUX'}
```

Comment:
the two query solution or one involving a subquery would also be nicer as it can use the department_id index.